docs/content/standards/display.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Sui Object Display
 description: The Sui Object Display standard is a template engine that enables on-chain management of off-chain representation (display) for a type.
+keywords: [ object display, object, display, display properties, object properties, sui::display, display<T>, Sui utility objects, Unique objects with dynamic representation, objects with unique static content ]
```

Comment:
I suspect `display<T>` might be causing the failed build. The `<T>` part. It definitely breaks when `<` or `{` are used in content without code fencing.